CONTRIBUTING.md

```diff
@@ -80,7 +80,7 @@ You can propose new designs for existing Docker features. You can also design
 entirely new features. We really appreciate contributors who want to refactor or
 otherwise cleanup our project. For information on making these types of
 contributions, see [the advanced contribution
-section](https://docs.docker.com/opensource/workflow/advanced-contributing/) in
+section](https://docs.docker.com/contribute/) in
```

Comment:
We may need to dig up these files; they're either in git history in this repository, or somewhere in https://github.com/docker/docs not sure where they originally were 🤔